### PR TITLE
Process variables using select

### LINKED ISSUE #41: Make the `vrb` argument in the `plot_*()` functions evaluate the input like e.g. `mutate()` does, instead of character/string input

### PR DESCRIPTION
**Pull request by pepijnvink**
Fixes #41

### CODE REVIEW

**Comment by pepijnvink** [comment]
Possibly solves #42 and #63 too

File: R/plot_corr.R [hunk 1]
```diff
@@ -19,7 +19,7 @@ plot_corr <- function(data, vrb = "all", label = FALSE, square = TRUE, diagonal
   if (vrb[1] == "all") {
     vrb <- names(data)
   } else {
-    vrb <- names(select(data, all_of({{vrb}})))
+    vrb <- names(dplyr::select(data, all_of({{vrb}})))
```
Comment (hanneoberman) [inline R/plot_corr.R:22]: Could you please add [`tidyselect::`](https://www.rdocumentation.org/packages/tidyselect/versions/1.2.0/topics/all_of) before all of the `all_of(...)` calls for clarity/completeness?

**Comment by hanneoberman** [comment]
Thanks! Unfortunately, this is not a universal fix. The package checks fail with the following error message:

```
   no applicable method for 'select' applied to an object of class "mids"
```

This makes sense, because a `mids` object is not a data frame and can therefore not be subsetted with data frame functions. Maybe it could be resolved by using the `select` function on the `$data` part of the `mids` object?

**Comment by hanneoberman** [comment]
Too bad, the fix did not yet work. Could you please look at `plot_trace()`? Specifically this line yields an error:

```
if (vrb %nin% varlist & vrb[1] != "all") {
``` 
Otherwise we'll investigate the problem together tomorrow!

File: R/ggmice.R [hunk 2]
```diff
@@ -36,6 +36,9 @@ ggmice <- function(data = NULL, mapping = ggplot2::aes()) {
   }
   mapping_x <- ggplot2::as_label(mapping$x)
   mapping_y <- ggplot2::as_label(mapping$y)
+  if (stringr::str_detect(mapping_x, "log\\(") | stringr::str_detect(mapping_y, "log\\(")){
+    stop("ggmice currently does not support log transformations in the mapping argument. Please transform the data before input, or use the scale_log() function.")
+  }
```
Comment (hanneoberman) [inline R/ggmice.R:41]: Nice! Only `scale_log()` does not seem to exist. Maybe replace by `  scale_*_continuous(trans='log10')`?

**Pull request merged**